features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2280,6 +2280,121 @@ FOR EACH ROW MODE DB2SQL
     END
 /
 
+CREATE TABLE AUTHORIZATION_DETAILS_TYPES (
+    ID CHAR(36) NOT NULL,
+    TYPE VARCHAR(255) NOT NULL,
+    API_ID CHAR(36) NOT NULL,
+    CURSOR_KEY INTEGER NOT NULL,
```

Comment:
Why we need this cursor key

Reply:
This field can be used in the "cursor pagination" which is more efficient than "limit/offset pagination"